rosidl_generator_py/cmake/rosidl_generator_py_get_typesupports.cmake

```diff
@@ -15,7 +15,7 @@
 macro(accumulate_typesupports)
   set(_typesupport_impl "")
   get_rmw_typesupport(_typesupport_impl ${rmw_implementation} LANGUAGE "C")
-  list(APPEND _typesupport_impls ${_typesupport_impl})
+  list_append_unique(_typesupport_impls ${_typesupport_impl})
```

Comment:
good point, the same macro is used in [generate_interfaces](https://github.com/ros2/rosidl/blob/unique_typesupport/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake) multiple times so I won't expect this to fail but we should fix it indeed